pkg/dashboard/batch.go

```diff
@@ -523,19 +523,14 @@ func (api *API) genPodDiffs(ctx context.Context, mountPods []corev1.Pod, shouldD
 		}
 	}
 	var secretList corev1.SecretList
-	ls := &metav1.LabelSelector{
-		MatchLabels: map[string]string{
-			common.JuicefsSecretLabelKey: "true",
-		},
-	}
-	s, _ := metav1.LabelSelectorAsSelector(ls)
-	if err := api.cachedReader.List(ctx, &secretList, &client.ListOptions{LabelSelector: s}); err != nil {
+	if err := api.cachedReader.List(ctx, &secretList, &client.ListOptions{}); err != nil {
```

Comment:
would be better to add a secret indexer in dashboard controller to avoid listing all each time